lib/assets/javascripts/cartodb/table/overlays/overlay_properties_bar.js

```diff
@@ -198,7 +198,13 @@ cdb.admin.OverlayPropertiesBar =  cdb.core.View.extend({
   render: function() {
 
     this._addForm();
-    this._addActions();
+
+    // disable 'Actions' edit section for fixed title/description added via map options
+    // (both report their type as 'header')
+    var modelType = this.model.attributes.type;
+    if (modelType != 'header') {
```

Comment:
It looks like theres a hide_actions parameter on the cdb.admin.MapOverlays._addOverlayPropertiesBar method. Wouldn't it make more sense to use that to hide the actions toolbar rather than a special case within the OverlayPropertiesBar class?